config/evss/mock_letters_response.yml.example

```diff
@@ -1,6 +1,6 @@
 ---
 # Mark Webb
-'796066621':
+'796104437':
```

Comment:
This may need to bring in some recent changes @kreek made to this file...LMK if you have questions. I think he made some updates to 1) allow a default response in case the logged in user isn't in this file, and 2) fixing an SSN/name pair